warehouse/admin/templates/admin/projects/detail.html

```diff
@@ -54,7 +54,13 @@ <h3 class="project-name text-center">{{ project.name }}</h3>
                 </tr>
                 <tr>
                   <td>Upload limit</td>
-                  <td>{{ project.upload_limit }}</td>
+                  <td>
+                    {% if project.upload_limit %}
+                      {{ project.upload_limit / ONE_MB }} MB ({{ "{:,.0f}".format(project.upload_limit) }} bytes)
```

Comment:
To format the bytes, I think it'd be better to use ``project.upload_limit|format_number()``, it's more comprehensible then the terrible Python mini language :D

I'm torn on the display of the MB itself, we have ``filesizeformat(binary=True)`` which will do the same thing, except work nicer if we go > 1GB... but I don't think that we're likely to do that so I don't know how much it matters. Since I'm already asking for you to fix the number formatting for the bytes, I think unless you feel strongly about it using ``{{ project.upload_limit|filesizeformat(binary=True) }}`` would be my preference.

Reply:
Done.